sda/processor.go

```diff
@@ -95,8 +93,7 @@ func (p *ServiceProcessor) RegisterMessages(procs ...interface{}) error {
 	return nil
 }
 
-// Process implements the Processor interface and dispatches ClientRequest message
-// and InterServiceMessage.
+// Process implements the Processor interface and dispatches ClientRequest messages.
 func (p *ServiceProcessor) Process(packet *network.Packet) {
```

Comment:
is it ever used really ? Does the messages going to the `ServiceProcessor` are not going through the serviceManager or something ?